CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Coding Style
+
+# Issues
+
+# Pull Request
+
+## Managing Submodule
+
+If your changes require libdds changes to be include same then there is a few
+extra steps to follow. You have to make sure required libdds changes have been
+merged to [http:://github.com/online-bridge-hackathon/libdds](http:://github.com/online-bridge-hackathon/libdds).
+
+Merging commits to libdds doesn't automatically include merges to the library
+used by service build process. The library version used by service is based on
+commit attached to the submodule state. To update the commit you need to first
+checkout the new commit into your submodule working tree. Then the change needs
+to be staged and committed like source code changes. Following example shows
```

Comment:
"The following example shows the"

Reply:
I feel like I can't say the example sentence in plain English. It seems like my thinking process is influenced by Finnish sentence structure which doesn't map 1:1 with English.

The latest attempt: The following example shows steps to update the submodule to the latest commit in the master branch.

I'm left with feeling there would be a simpler way to say same in English.

Reply:
I know how that goes. I often have the same problem, and English is my native language!

I think your latest version is fine. Slightly better might be:

"The following example shows the steps to update the submodule to the latest commit in the master branch."